src/plot_api/plot_api.js

```diff
@@ -494,7 +494,7 @@ function setPlotContext(gd, config) {
     context._hasZeroWidth = context._hasZeroWidth || gd.clientWidth === 0;
 }
 
-function plotPolar(gd, data, layout) {
+function plotLegacyPolar(gd, data, layout) {
```

Comment:
Good call renaming this function :trophy: 

Looks like legacy polar's `layout.radialaxis` did not support a `title`, so :ok_hand: